airflow/config_templates/config.yml

```
@@ -979,6 +979,22 @@ logging:
       type: boolean
       example: ~
       default: "True"
+    color_log_error_keywords:
```

Not to be too "picky" but you name the config arguments "error" and "warning" - mainly because you apply typical red/yellow colors as highlight. So in most cases this really apply to errors and warnings. And this is common. But functional wise it is not linked to this. Did you consider naming the config keys `color_log_red_highlight` and `color_log_yellow_highlight`? (and of course keeping error/warn as default values) It would be a bit closer to what actually is configured.
```suggestion
    color_log_red_highlight_keywords:
```

---

I don't want to add color names of red to the config option in case it changes before release or so. I guess for me color_log_error_keywords is good enough.